### DISCUSSION TITLE
Moving the document to markdown

### DISCUSSION THREAD

**Discussion by maryjom** [w3c/wcag2ict#55] [Ideas]
To continue the discussion started on the Welcome thread via these main comments (and replies):
- [Bruce's initial question about using markdown](https://github.com/w3c/wcag2ict/discussions/9#discussioncomment-3919551)
- [Bruce's comment providing a pointer to Section508.gov and github repository](https://github.com/w3c/wcag2ict/discussions/9#discussioncomment-3925540)

I've played with the GitHub wiki markdown, and found that no amount of code coaxing enabled me to make any text color changes or background color changes when using the Wiki Markdown. I tried using a span with a color, some CSS, and all gets stripped away by GitHub's wiki code. 

This means the Wiki templates for adding SC's are all black and white text. There's only a gray line you can add that spans the text area that you can invoke by having a line with three asterisks in a row. For now, I've used that markdown to denote two areas of instructions. So this is not the kind of markdown we would ultimately want to use.

**Comment by bruce-usab** [comment]
Markdown can reference pre-defined classes which I am sure @michael-n-cooper has more than a few we can use.

My own experience is that the limitations, like **_not_** being able to color changes, is a huge plus because it provokes a conversation about such as:  Why does this paragraph need to be big and red?  But I can understand that it might be helpful to visually separate instructions for working on content from the actual content.

Semantic stuff, for example definition lists and <ins>insertions</ins> and <del>deletions</del> are probably supported.

**Comment by maryjom** [comment]
The document has been moved to markdown. There are still formatting issues we're dealing with, and extremely limited pre-defined classes (Editor's note, Note, and Example). I am asking for one for WCAG2ICT guidance, and we'll see if they give us a special "callout" class for it. There's also issues with section numbering and inability to turn on/off that capability for certain sections (or areas of sections) so the section numbers can align with WCAG's numbering. My what a tangled web we weave.... I'll close this discussion, as we have moved to markdown and are tracking any problems with GitHub issues.
